code/datalab-app/docker/config/traefik/traefik.toml

```diff
@@ -0,0 +1,33 @@
+# traefik.toml
+logLevel = "DEBUG"
+accessLogsFile = "/logs/access.log"
+defaultEntryPoints = ["http"]
+[entryPoints]
+  [entryPoints.http]
+  address = ":80"
+
+[web]
+address = ":8080"
+
+[file]
+
+# rules
+[backends]
+  [backends.files]
+    [backends.files.servers.server1]
+    url = "http://minio:9000"
+    weight = 1
+  [backends.files-connect]
+    [backends.files-connect.servers.server1]
+    url = "http://10.1.24.116:8001"
```

Comment:
This isn't but I couldn't figure out how to get the container to reference back to the host. I tried localhost, 0.0.0.0 and 127.0.0.1 but these are all obviously directing to the container. I think the problem goes away when we separate into a different container.